test/boulder-test-config.json

```diff
@@ -32,12 +32,10 @@
 
   "wfe": {
     "baseURL": "http://localhost:4300",
-    "listenAddress": "127.0.0.1:4300"
+    "listenAddress": "0.0.0.0:4300"
```

Comment:
Let's please not do this; it violates firewall rules on locked down Linuxes and BSDs.